src/NuGet/Microsoft.CodeAnalysis.CSharp.Workspaces.nuspec

```diff
@@ -27,6 +27,8 @@ Supported Platforms:
     <file src="Microsoft.CodeAnalysis.CSharp.Workspaces.xml" target="lib\portable-net45+win8" />
     <file src="Microsoft.CodeAnalysis.CSharp.Workspaces.dll" target="lib\net45" />
     <file src="Microsoft.CodeAnalysis.CSharp.Workspaces.xml" target="lib\net45" />
+    <file src="Microsoft.CodeAnalysis.CSharp.Workspaces.dll" target="lib\netstandard1.3" />
```

Comment:
This is also already how it works in Microsoft.CodeAnalysis.CSharp. Why is it OK there but not OK here?